test/error-placement.js

```diff
@@ -291,6 +291,42 @@ test( "test existing non-label used as error element", function(assert) {
 	assert.noErrorFor( field );
 });
 
+test( "test aria-describedby with input names contains special characters (:, ., [, ], $)", function() {
```

Comment:
Great, lets support everything jQuery supports (so the charlist you mentioned above). I dont think we need to be 100% compilant with all those unicode special chars etc.